kalite/coachreports/static/js/coachreports/coach_reports/models.js

```diff
@@ -33,6 +33,18 @@ var GroupCollection = Backbone.Collection.extend({
     url: window.GROUP_RESOURCE_URL
 });
 
+function normalizeEndDate(end_date) {
```

Comment:
I think we were trying to use underscore-casing for everything. But not a blocker.